Tests/OAuth/ResourceOwner/GitHubResourceOwnerTest.php

```diff
@@ -50,4 +52,31 @@ protected function setUpResourceOwner($name, $httpUtils, array $options)
     {
         return new GitHubResourceOwner($this->buzzClient, $httpUtils, $options, $name, $this->storage);
     }
+
+    public function testCustomResponseClass()
+    {
+        $this->buzzCalls = 2;
+
+        parent::testCustomResponseClass();
+
+        $this->buzzCalls = 1;
```

Comment:
testCustomResponseClass() and testGetUserInformation() trigger two API calls, but the Buzz mock is hardcoded to expect just one call. I've slightly modified it to be "configurable", just for the github tests.